internal/metrics/metrics.go

```diff
@@ -249,35 +250,44 @@ var (
 		[]string{"chain_name", "upstream_id", "url", "method"},
 	)
 
-	errorLatencyStatusCheckRequests = promauto.NewCounterVec(
+	errorLatencyStatusCheckErrors = promauto.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: metricsNamespace,
 			Subsystem: "healthcheck",
-			Name:      "latency_check_requests",
-			Help:      "Total latency check requests made.",
+			Name:      "error_check_has_errors",
+			Help:      "Number of errors of upstream requests.",
 		},
-		[]string{"chain_name", "upstream_id", "url", "method"},
+		[]string{"chain_name", "upstream_id", "url", "errorType", "method"},
 	)
 
-	errorLatencyStatusCheckDuration = promauto.NewHistogramVec(
-		prometheus.HistogramOpts{
+	errorLatencyStatusCheckNoErrors = promauto.NewCounterVec(
+		prometheus.CounterOpts{
 			Namespace: metricsNamespace,
 			Subsystem: "healthcheck",
-			Name:      "latency_check_duration_seconds",
-			Help:      "Latency of latency check requests.",
-			Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 20, 40},
+			Name:      "error_check_no_errors",
+			Help:      "Number of no errors of upstream requests.",
 		},
-		[]string{"chain_name", "upstream_id", "url", "method"},
+		[]string{"chain_name", "upstream_id", "url", "errorType", "method"},
 	)
 
-	errorLatencyStatusCheckErrors = promauto.NewCounterVec(
+	errorLatencyStatusCheckErrorsIsPassing = promauto.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: metricsNamespace,
 			Subsystem: "healthcheck",
-			Name:      "latency_check_errors",
-			Help:      "Errors of upstream requests.",
+			Name:      "error_check_is_passing",
+			Help:      "Number of passing error checks.",
 		},
-		[]string{"chain_name", "upstream_id", "url", "errorType", "method"},
+		[]string{"chain_name", "upstream_id", "url", "errorType"},
```

Comment:
I think we need to remove `errorType` from the labels beacuse we're not emitting it anymore. Having this label will result in wrong labels because I believe labels are matched up by position.

Reply:
Ooh nice one sorry for sending you down that rabbit hole